app/Resources/views/base.html.twig

```diff
@@ -43,7 +43,7 @@
                             <a href="{{ path('ra_management_manage') }}">{{ 'ra.menu.ra_management'|trans }}</a>
                         </li>
                     {% endif %}
-                    {% if showsRaLocations %}
+                    {% if institution_configuration_options.getInstitutionConfigurationOptions.useRaLocations %}
```

Comment:
Why the "extra" accessor and not just `institution_configuration_options.useRaLocations`?

Secondary, it seems a bit inconsisten with the snake_cased_var.camelCasedProperty - how about camelcasing the var as well?